product_dimension/migrations/12.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 C2i Change 2 improve - Eduardo Magdalena <emagdalena@c2i.es>
```

Comment:
The directory name that containt the file is wrong.
Right now is "12.0.1.0.0" it should be "12.0.2.0.0" it the number of the new version

Reply:
ok, changed